index.html

```diff
@@ -38,162 +44,182 @@ <h3 class="cover-links">LET'S CONNECT</h3>
+            </p>   
+            <button class="tag1">HTML</button>
+            <button class="tag2">CSS</button>
+            <button class="tag3">Javascript</button>
+          
+            <button class="project-detail">See Project</button>
+        </div>
+      </section>
+  
+      <!-- Second project -->
+      <section class="work-page-left">
+        <div class="card2-image">
+          <img
+          src="images/works/multi_post_story/cover.svg"
+          alt="work1"
+          width="100%"
+        />
+        </div>
+        <div class="card2-content">
+            <h2 class="work-title">Tonic</h2>
+            <ul class="clients">
+              <li>CANOPY</li>
+              &nbsp;
+              <img
+                src="images/works/Counter.svg"
+                alt="counter"
+                class="counter"
+              />&nbsp;
+              <li>Back</li>
+              &nbsp;
+              <li>End</li>
+              &nbsp;
+              <li>Dev</li>
+              &nbsp;
+              <img
+                src="images/works/Counter.svg"
+                alt="counter"
+                class="counter"
+              />&nbsp;
+              <li>2015</li>
+            </ul>
+            <p class="highlights">
+              A daily selection of privately personalized reads no accounts for
+              sign-ups required.
+            </p>
+            <button class="tag1">HTML</button>
+            <button class="tag2">CSS</button>
+            <button class="tag3">Javascript</button>
+          
+            <button class="project-detail">See Project</button>
+        </div>
+      </section>
 
-    <!-- About myself -->
+        <!-- Third project -->
+      <section class="work-page-right">
+       <div class="card3-image">
+        <img src="images/works/tonic/cover1.svg" alt="work1" width="100%" />
+       </div>
+        <div class="card3-content">
+              <h2 class="work-title">Tonic</h2>
+            <ul class="clients">
+              <li>CANOPY</li>
+              &nbsp;
+              <img
+                src="images/works/Counter.svg"
+                alt="counter"
+                class="counter"
+              />&nbsp;
+              <li>Back</li>
+              &nbsp;
+              <li>End</li>
+              &nbsp;
+              <li>Dev</li>
+              &nbsp;
+              <img
+                src="images/works/Counter.svg"
+                alt="counter"
+                class="counter"
+              />&nbsp;
+              <li>2015</li>
+            </ul>
+            <p class="highlights">
+              A daily selection of privately personalized reads no accounts for
+              sign-ups required.
+            </p>   
+              <button class="tag1">HTML</button>
+              <button class="tag2">CSS</button>
+              <button class="tag3">Javascript</button>
+  
+            <button class="project-detail">See Project</button>
+        </div>
+      </section>

```

Comment:
- Good job done so far 👏. Please add more spacing between the cards so it looks close to how it does in the Figma template.  Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project 👇:
<img width="828" alt="sad" src="https://user-images.githubusercontent.com/73607512/199807238-23541f9e-5d45-49dc-ab67-3a8c68f1c905.PNG">



Figma Design:
<img width="487" alt="gjgjg" src="https://user-images.githubusercontent.com/73607512/199807084-0640e352-e6b2-4cb5-85a4-491bf0eeb61b.PNG">

Reply:
- [ ] As suggested by the previous code reviewers, @mckent05 and @Favourezeugwa, Kindly you need to add more space between cards to match that of Figma design, :joy: 
Please try to simply modify the margin-top of your ```.work-page-left``` style to 👉🏽 ```margin: 4rem 0;``` on desktop screen. as this will make the ut the suggested space between the cards, You can follow this [link ](https://www.w3schools.com/css/css_margin.asp)for more details on CSS margins :pray: